src/posthog-core.ts

```diff
@@ -787,7 +793,11 @@ export class PostHog {
      * @param {String} [config.transport] Transport method for network request ('XHR' or 'sendBeacon').
      * @param {Date} [config.timestamp] Timestamp is a Date object. If not set, it'll automatically be set to the current time.
      */
-    capture(event_name: string, properties?: Properties | null, options?: CaptureOptions): CaptureResult | undefined {
+    capture(
+        event_name: EventName,
+        properties?: Properties | null,
+        options?: CaptureOptions
```

Comment:
```suggestion
        options?: CaptureOptions | null
```

do we need this here too?

Reply:
I don't _think_ so... I'll pull on the thread and see if anything is sending null but I didn't change that here iirc so 🤞